### ISSUE TITLE
Time-domain interface - [opened]

### ISSUE THREAD

**Issue by mj-will** [mj-will/bilby#27]
In GitLab by @ecm on Jun 16, 2022, 14:34

_Merges master -> master_

Last Feb, Tom Colin and I presented a work aiming at interfacing Bilby to time-domain CBC waveform models (with the final aim to use custom models from ML-based approximants). See: https://dcc.ligo.org/LIGO-G2200091

We are now ready to make a merge request.

**Comment by mj-will** [comment]
In GitLab by @gregory.ashton on Jun 17, 2022, 10:19

Hi @ecm thanks for the MR. There is quite a lot of additions to review, so this may take some time, but at q quick check it looks like it is in the write style.

The one thing I'm puzzled about it that I can't see the CI status (i.e. the green tick). We need that passing to merge. Is it because your fork is private?

**Comment by mj-will** [comment]
In GitLab by @ecm on Jun 23, 2022, 09:25

Hi @gregory.ashton 

thank you for the feedback. This is the first time I do a MR. I was pointed 
to a page here: https://git.ligo.org/lscsoft/bilby/-/issues/627
I've tried to follow the steps but it is fairly probable that I did something
wrong. Yes, the forked project is private. https://git.ligo.org/ecm/bilby
Are you saying I should change something in the config of my repo to enable
CI ?

Cheers,
Eric
